odoo/addons/base/views/res_partner_views.xml

```diff
@@ -102,7 +102,7 @@
                         <field name="company_type" options="{'horizontal': true}" widget="radio" groups="base.group_no_one"/>
                         <h1>
                             <field id="company" name="name" default_focus="1" placeholder="e.g. Lumber Inc" invisible="not is_company" required="type == 'contact' and is_company"/>
-                            <field id="individual" name="name" default_focus="1" placeholder="e.g. Brandom Freeman" invisible="is_company" required="type == 'contact' and not is_company"/>
```

Comment:
Can we please split the changes in the res.partner form view in a separate commit?

Not really related to the web JS framework improvements.